src/view/event-edit-view.js

```diff
@@ -1,167 +1,162 @@
 import {createElement} from '../render.js';
+import {humanizeEventDate} from '../utils.js';
+import {offersСatalog} from '../fish/offers.js';
+
+const getOffersByType = (type) => offersСatalog.find((offer) => offer.type === type);
+
+const createOfferTemplate = (currentOffer, selectedOffers) => {
+  const {id, title, price} = currentOffer;
+  const isAlreadySelected = selectedOffers.find((selected) => title === selected.title);
+
+  return (
+    `<div class="event__offer-selector">
+      <input class="event__offer-checkbox  visually-hidden"
+             id="event-offer-${id}"
+             type="checkbox"
+             name="event-offer"
+             value="${id}"
+             ${isAlreadySelected ? 'checked' : ''}>
+      <label class="event__offer-label" for="event-offer-${id}">
+        <span class="event__offer-title">${title}</span>
+          &plus;&euro;&nbsp;
+        <span class="event__offer-price">${price}</span>
+      </label>
+    </div>`
+  );
+};
+
+const createOffersTemplate = (availableOffers, selectedOffers) => {
+  if (availableOffers.length === 0) {
```

Comment:
Офферов может не быть, лучше проверить с помощью https://developer.mozilla.org/ru/docs/Web/JavaScript/Reference/Operators/Optional_chaining
```suggestion
  if (availableOffers?.length === 0) {
```

Reply:
принято